bjoern/request.c

```diff
@@ -174,24 +182,25 @@ on_headers_complete(http_parser* parser)
 static int
 on_body(http_parser* parser, const char* data, const size_t len)
 {
-  Iobject* body;
+  PyObject *body;
 
-  body = (Iobject*)PyDict_GetItem(REQUEST->headers, _wsgi_input);
+  body = PyDict_GetItem(REQUEST->headers, _wsgi_input);
   if(body == NULL) {
     if(!parser->content_length) {
       REQUEST->state.error_code = HTTP_LENGTH_REQUIRED;
       return 1;
     }
-    PyObject* buf = PyString_FromStringAndSize(NULL, parser->content_length);
-    body = (Iobject*)PycStringIO->NewInput(buf);
-    Py_XDECREF(buf);
-    if(body == NULL)
-      return 1;
-    _set_header(_wsgi_input, (PyObject*)body);
-    Py_DECREF(body);
+    body = PyObject_CallMethodObjArgs(IO, STR_bytesio, NULL);
+    if (body == NULL) {
+	    printf("call io.BytesIO failed\n");
```

Comment:
Get rid of it. as it has a none zero return value. Also changed another place printf()->PyErr_SetString().